RecoJets/JetProducers/plugins/VirtualJetProducer.cc

```diff
@@ -892,3 +843,55 @@ void VirtualJetProducer::writeCompoundJets(  edm::Event & iEvent, edm::EventSetu
   }
 
 }
+
+// ------------ method fills 'descriptions' with the allowed parameters for the module  ------------
+void VirtualJetProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+
+	edm::ParameterSetDescription desc;
+        fillDescriptionsFromVirtualJetProducer(desc);
+	desc.add<string>("jetCollInstanceName", ""	);
+	descriptions.addDefault(desc);
+}
+
+void VirtualJetProducer::fillDescriptionsFromVirtualJetProducer(edm::ParameterSetDescription& desc)
+{
+	desc.add<string> ("@module_label",	"" );
```

Comment:
indeed this should not even be a valid parameter (because the frmework uses it internally for the module's label)